scs2-simulation/src/main/java/us/ihmc/scs2/simulation/physicsEngine/contactPointBased/ContactPointBasedPhysicsEngine.java

```diff
@@ -117,8 +117,7 @@ public void simulate(double currentTime, double dt, Vector3DReadOnly gravity)
 
       for (ContactPointBasedRobot robot : robotList)
       {
-         robot.computeJointDamping();
-         robot.computeJointSoftLimits();
+         robot.computeJointSimulationEffects();
```

Comment:
I've coupled these methods together, as they're only ever called together... right?

Reply:
Sounds good! Maybe let's call that `computeJointLowLevelControl()`?